recipes/angsd/meta.yaml

```diff
@@ -1,15 +1,13 @@
-{% set version = "0.939" %}
-{% set sha256 = "43bbc02d964eea59508230a63f44b43d58be5d7ef450064ce5eebd0b0ba3b9dc" %}
```

Comment:
You downgraded the version?

Reply:
Hi,

I am in the ANGSD development team and I am a PhD student working with Thorfinn Korneliussen (who is the main developer of ANGSD). The stable release tarballs of ANGSD are the version numbers that are even, and mainly distributed from popgen.dk website. Therefore we would like to update (downgrade) the angsd bioconda recipe to the latest stable release (0.938) and the link to the main website (popgen.dk). Thanks!

Best regards,
Isin

Reply:
@isinaltinkaya that makes sense. But people installing angsd without a version will still get the latest, e.g. the beta version. So maybe you can bump the version on github once and then increase the number here?

Reply:
I am not sure if I understand- can you explain what you mean by "bump the version on github once and then increase the number here?"? Thanks!

Reply:
@bgruening we have just released 0.940 which I included in this PR. Is there anything else I should do?